openghg/store/_obscolumn.py

```diff
@@ -115,6 +116,8 @@ def read_file(
         site = clean_string(site)
         satellite = clean_string(satellite)
         domain = clean_string(domain)
+        region = clean_string(region)
+        selection = clean_string(selection)
         network = clean_string(network)
         instrument = clean_string(instrument)
```

Comment:
Suggestion to add a relationship between `domain` and [`obs_`]`region` as follows:
- if `domain` is specified, `obs_region` is not specified (None) - update `obs_region` to `match` domain and write a message to the log.
- if `domain` AND `obs_region` are specified, can raise an error and include an error log message to indicate only one should be included.
- if `obs_region` is specified, `domain` is not specified, `domain` can be set to the keyword default (I think that should be "NOT_SET" at the moment. Linked to https://github.com/openghg/openghg/blob/devel/openghg/util/_metadata_util.py#L37)